### PR TITLE
Added Group `ping`

### PR DESCRIPTION
**Pull request by RichardLitt**
Pretty straightforward

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1690,6 +1690,238 @@ root will not be listable, as it is virtual. Access known paths directly.
 ## rm
 
 # Group ping
+Send echo request packets to IPFS hosts
```
Comment (dignifiedquire) [inline apiary.apib]: Missing `.`

File: apiary.apib [hunk 2]
```diff
@@ -1690,6 +1690,238 @@ root will not be listable, as it is virtual. Access known paths directly.
...
+
+## ping [GET /ping{?arg}{&count}]
+A tool to test sending data to other nodes. It finds nodes
+via the routing system, send pings, wait for pongs, and print out round-
```
Comment (dignifiedquire) [inline apiary.apib]: send**s** pings

Comment (dignifiedquire) [inline apiary.apib]: wait**s** for pongs and print**s** out

**Comment by RichardLitt** [comment]
@dignifiedquire Fixed with updated comments. RFM?

**Comment by dignifiedquire** [comment]
LGTM

**Comment by RichardLitt** [comment]
Thanks, @dignifiedquire!

**Pull request merged**